dashboard/src/utils/axiosInstance.js

```diff
@@ -30,7 +28,8 @@ axiosInstance.interceptors.response.use(
   (error) => {
     if (error?.response) {
       const { status } = error.response;
-      if (status === UNAUTHORIZED) {
+      const { responseURL } = error.request;
+      if (status === UNAUTHORIZED && !responseURL.includes("login")) {
         dispatch(showSessionExpired());
       }
```

Comment:
So, _any_ response of "unauthorized" from a URI which doesn't include `login` will be deemed to indicate an expired session?  That might be true, if the Dashboard is highly disciplined, but I'm having trouble imagining how we'll cover all the use cases without ever violating that idea.

That is, I expect that there will be cases where a given user can request a particular result or view, and, for various reasons, the server will respond with `UNAUTHORIZED`...not because the user's session has expired but rather because the requesting user is specifically not authorized to access the target resource (e.g., because the data belongs to another user who has not published it).

So, I think we're going to find that this criterion is insufficient.

Reply:
As clarified in today's Dashboard Sync meeting, we expect that `FORBIDDEN` will be returned for access failures, while `UNAUTHORIZED` will be reserved for token problems, so this code should be OK as is (however, Dave suggested replacing the URL test with a different criterion tied to whether the request had specified a token).

Reply:
@dbutenhof, I'm going to leave it to you to resolve this conversation:  you mentioned that the criterion in the code should be changed; I don't know whether you were hoping to see that in this PR or not.